src/check.js

```diff
@@ -0,0 +1,123 @@
+/* eslint no-use-before-define: ["error", { "variables": false }] */
```

Comment:
- [ ] Kindly remove the eslint disable rule. It's best to fix the issues in the code.